tests/requirements/base.txt

```diff
@@ -6,6 +6,8 @@ jinja2 >= 2.7
 numpy
 Pillow
 PyYAML
+# Remove if it ends up being possible to build libmemcached on the Windows Jenkins box.
```

Comment:
I'm not happy to merge the comment. Let's get an answer on whether or not this is possible.